sdk/ai/azopenai/CHANGELOG.md

```diff
@@ -1,9 +1,11 @@
 # Release History
 
-## 0.5.1 (Unreleased)
+## 0.6.0 (Unreleased)
```

Comment:
Shouldn't this remain `0.5.1`? I don't see any breaking changes.

Reply:
Dimensions is a new feature so just going with that. 

Is it not <major>.<minor>.<patch> still, even for betas? Or do you think of it more like 0.<major>.<minor>?

Reply:
For betas we've been following `0.<major>.<minor>` although I don't think we've been super-stringent on that.

Reply:
Ah, okay, didn't realize we had a convention there. I'll bump it back :)